addons/mail/static/src/discuss/call/common/call_invitations.scss

```diff
@@ -1,3 +1,7 @@
 .o-discuss-CallInvitations {
     z-index: $zindex-modal;
 }
+
+.o-discuss-CallInvitation-avatar {
+    aspect-ratio: 1;
+}
```

Comment:
Bootstrap has [`aspect-ratio` class](https://getbootstrap.com/docs/5.3/helpers/ratio/#aspect-ratios) available, we should use them instead.

Reply:
@phenix-factory I already tried it first with `ratio ratio-1x1`, it is to be used in a wrapper element so there is more diff to add the wrapper itself and get the aligment righ (it generates a pseudo element (`::before`) which makes it difficult).

Reply:
Well I guess it's fine this way then